### ISSUE TITLE
输出的一条警告

### ISSUE THREAD

**Issue by g2bond** [Q-Future/Visual-Question-Answering-for-Video-Quality-Assessment#7]
你好，
我在运行python ./llava/eval/model_conv.py后输出了一条像语句的警告，在我用***标记的位置：
You will receive 4 distinct frames that have been by uniformly sampling 1 frame per second from the video, arranged in the same temporal order as they appear in the video. In addition, you will also obtain motion features extracted from all 92 frames of the the first video. Please analyze these images and answer the questions based on your observations.
The video frames:<image>
The video motion features:<image>
How is the flickers of the video?
***The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.***
answer:There are two instances of flicker: one beginning at 2 seconds with a duration of 1 second, and another starting at 3 seconds, also lasting 1 second.
请问这个是正常的吗，或者我需要改哪里。我用的是q-future/VQA-Assistant-llava-qwen-enhanced模型

**Comment by jzhws** [comment]
正常的
